### ISSUE TITLE
Webapp is live, but is pinging wrong mongo cluster

### ISSUE THREAD

**Issue by W-A-James** [gbuchdahl/contagion-tracker#21]
Since our mongo accounts got closed the map doesn't really function anymore. @gbuchdahl do you think you could get the new srv string from the new cluster and push that to the heroku deployment?

**Comment by gbuchdahl** [comment]
Yes - @nalin1729 @W-A-James can one of you add my gbuchdahl [at] gmail email to the atlas instance?

**Comment by nalin1729** [comment]
@gbuchdahl Sent the invites, lmk if there are issues
